python/doc/examples/meta_modeling/kriging_metamodel/plot_gpr_1d.py

```diff
@@ -0,0 +1,284 @@
+#
+# We begin by defining the `alpha` variable containing the complementary of the confidence level than we want to compute.
+# Then we compute the quantile of the Gaussian distribution corresponding to `1-alpha/2`. Therefore, the confidence interval is:
+#
+# .. math::
+#    P\in\left(X\in\left[q_{\alpha/2},q_{1-\alpha/2}\right]\right)=1-\alpha.
+#
+#
+
+# %%
+alpha = 0.05
+
+
+def computeQuantileAlpha(alpha):
+    bilateralCI = ot.Normal().computeBilateralConfidenceInterval(1 - alpha)
+    return bilateralCI.getUpperBound()[0]
+
+
+quantileAlpha = computeQuantileAlpha(alpha)
+print("alpha=%f" % (alpha))
+print("Quantile alpha=%f" % (quantileAlpha))
+
+# %%
+# In order to compute the regression error, we can consider the conditional variance.
+# The `getConditionalCovariance` method returns the covariance matrix `covGrid`
+# evaluated at each points in the given sample. Then we can use the diagonal
+# coefficients in order to get the marginal conditional Kriging variance.
+# Since this is a variance, we use the square root in order to compute the
+# standard deviation.
+# However, some coefficients in the diagonal are very close to zero and
+# nonpositive, which leads to an exception of the sqrt function.
+# This is why we add an epsilon on the diagonal (nugget factor), which prevents this issue.
+
+# %%
+sqrt = ot.SymbolicFunction(["x"], ["sqrt(x)"])
+epsilon = ot.Sample(n_test, [1.0e-8])
+gccc = otexp.GaussianProcessConditionalCovariance(gpr_result)
+conditionalVariance = gccc.getConditionalMarginalVariance(x_test) + epsilon
+conditionalSigma = sqrt(conditionalVariance)
+
+# %%
+# The following figure presents the conditional standard deviation depending on :math:`x`.
+
+# %%
+graph = ot.Graph(
+    "Conditional standard deviation", "x", "Conditional standard deviation", True, ""
+)
+curve = ot.Curve(x_test, conditionalSigma)
+graph.add(curve)
+view = viewer.View(graph)
+
+
+# %%
+# We now compute the bounds of the confidence interval. For this purpose we define a small function
+# `computeBoundsConfidenceInterval` :
+
+
+# %%
+def computeBoundsConfidenceInterval(quantileAlpha):
+    dataLower = [
+        [y_test_MM[i, 0] - quantileAlpha * conditionalSigma[i, 0]]
+        for i in range(n_test)
+    ]
+    dataUpper = [
+        [y_test_MM[i, 0] + quantileAlpha * conditionalSigma[i, 0]]
+        for i in range(n_test)
+    ]
+    dataLower = ot.Sample(dataLower)
+    dataUpper = ot.Sample(dataUpper)
+    return dataLower, dataUpper
+
+
+# %%
+# We define two small lists to draw three different confidence intervals (defined by the alpha value) :
+alphas = [0.05, 0.1, 0.2]
+# three different green colors defined by HSV values
+mycolors = [[120, 1.0, 1.0], [120, 1.0, 0.75], [120, 1.0, 0.5]]
+
+# %%
+# We are ready to display all the previous information and the three confidence intervals we want.
+
+# %%
+# sphinx_gallery_thumbnail_number = 4
```

Comment:
In this graph, the handle of transparency could improve the graph. Currently, we do not see the data points and the exact function

Reply:
Here is a function that may help you to do this.

```python
def toTransparentColor(color, alpha_transparency=128):
    """
    Convert a full color to a given color with transparency.

    Parameters
    ----------
    color : str
        The Hex color.
    alpha_transparency : int
        The alpha channel of the [R, G, B, A] color space

    Returns
    -------
    alpha_color : str
        The Hex color, with transparency.
    """
    [r, g, b] = ot.Drawable.ConvertToRGB(color)
    alpha_color = ot.Drawable.ConvertFromRGBA(r, g, b, alpha_transparency)
    return alpha_color
```

Reply:
yes I handle it by just inverting the drawables

Reply:
Maybe it could be nice to add somewhere (in the changelog or in the doc of GPR) a paragraph to explain what are the new functionalities given by the new classes related to GPR with respect to the legacy KrigingAlgorithm. This could be useful to help the user to these new classes (and explain why these new classes have been created).

Reply:
ok